### ISSUE TITLE
Topgrade don't upgrade npm app

### ISSUE THREAD

**Issue by PrzemekSkw** [r-darwish/topgrade#1022]
## What did you expect to happen?
upgrade npm installed app - Electerm

## What actually happened?
nothing happen

## Additional Details
- Which operating system or Linux distribution are you using? Garuda (Archlinux)
- How did you install Topgrade? AUR
- Which version are you running? `9.0.1` 

Run `topgrade --dry-run` to see which commands Topgrade is running.
```
 ╰─λ topgrade --dry-run

―― 20:19:26 - System update ――――――――――――――――――――――――――――――――――――――――――――――――――――
no new news
Dry running: /usr/bin/paru --pacman pacman -Syu --devel

―― 20:19:27 - TLDR ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
Dry running: /usr/bin/tldr --update

―― 20:19:27 - Gnome Shell extensions ―――――――――――――――――――――――――――――――――――――――――――
Dry running: /usr/bin/gdbus call --session --dest org.gnome.Shell.Extensions --object-path /org/gnome/Shell/Extensions --method org.gnome.Shell.Extensions.CheckForUpdates

―― 20:19:27 - micro ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
Dry running: /usr/bin/micro -plugin update

―― 20:19:27 - Flatpak User Packages ――――――――――――――――――――――――――――――――――――――――――――
Dry running: /usr/bin/flatpak update --user -y

―― 20:19:27 - Flatpak System Packages ――――――――――――――――――――――――――――――――――――――――――
Dry running: /usr/bin/flatpak update --system -y

―― 20:19:27 - Firmware upgrades ――――――――――――――――――――――――――――――――――――――――――――――――
Dry running: /usr/bin/fwupdmgr refresh
Dry running: /usr/bin/fwupdmgr get-updates

―― 20:19:27 - Summary ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
System update: OK
TLDR: OK
Gnome Shell Extensions: OK
Flatpak: OK
Firmware upgrades: OK

Pacman backup configuration files found:
/etc/pacman.d/mirrorlist.pacnew
/etc/paru.conf.pacnew
/etc/locale.gen.pacnew
```

Regards.
